test/AISmart.Application.Tests/GAgent/CqrsTest.cs

```diff
@@ -0,0 +1,55 @@
+using System;
+using System.Linq;
+using System.Threading.Tasks;
+using AISmart.Agent;
+using AISmart.Agent.Events;
+using AISmart.CQRS.Provider;
+using Orleans;
+using Shouldly;
+using Xunit;
+using Xunit.Abstractions;
+
+namespace AISmart.GAgent;
+
+public class CqrsTests : AISmartApplicationTestBase
+{
+    private readonly IClusterClient _clusterClient;
+    private readonly ITestOutputHelper _output;
+    private readonly ICQRSProvider _cqrsProvider;
+
+    public CqrsTests(ITestOutputHelper output)
+    {
+        _clusterClient = GetRequiredService<IClusterClient>();
+        _output = output;
+        _cqrsProvider = GetRequiredService<ICQRSProvider>();
+    }
+
+    [Fact]
+    public async Task SendTransactionTest()
+    {
+        string chainId = "AELF";
+        string senderName = "Test";
+        string address = "JRmBduh4nXWi1aXgdUsj5gJrzeZb2LxmrAbf7W99faZSvoAaE";
+        var createTransactionEvent = new CreateTransactionEvent()
+        {
+            ChainId = chainId,
+            SenderName = senderName,
+            ContractAddress = address,
+            MethodName = "Transfer",
+        };
+        var guid = Guid.NewGuid();
+        await _clusterClient.GetGrain<IAElfAgent>(guid).ExecuteTransactionAsync(createTransactionEvent);
```

Comment:
Use clusterfixture instead?

Reply:
At present, other single tests call grain in the same way. Is it necessary to change the calling method